src/Modules/SharedModule/components/NavBar/NavBar.tsx

```diff
@@ -46,61 +50,72 @@ export default function NavBar() {
-                        type="button"
-                        className=" border-0 bg-transparent "
-                        data-bs-toggle="dropdown"
-                        aria-expanded="false"
-                      >
-                        <i className="fa-solid fa-chevron-down text-theme"></i>
-                      </button>
-                      <ul className="dropdown-menu">
-                        <li>
-                          <a className="dropdown-item" href="#">
-                            Profile
-                          </a>
-                        </li>
-                        <li>
-                          <a className="dropdown-item" href="#">
-                            logOut
-                          </a>
-                        </li>
-                      </ul>
-                    </div>
+                <div className="me-2">
+                  <div className="btn-group dropstart">
+                    <button
+                      type="button"
+                      className=" border-0 bg-transparent "
+                      data-bs-toggle="dropdown"
+                      aria-expanded="false"
+                    >
+                      <div className="user-data d-flex  align-items-center ">
+                        <div className="user-img rounded-circle me-3">
+                          <img
+                            className="w-100 rounded-circle me-1"
+                            src={Images.test}
+                            alt=""
+                          />
+                        </div>
+                        <div className="user-info me-2 pt-2">
+                          <h6 className="text-theme mb-0 pb-0">
+                            {adminData?.userName}
+                          </h6>
+                          <p className="text-theme mb-0 pb-0">
+                            {adminData?.userEmail}
+                          </p>
+                        </div>
+                        <div>
+                          <i className="fa-solid fa-chevron-down text-theme"></i>
+                        </div>
+                      </div>
+                    </button>
+                    <ul className="dropdown-menu">
+                      <li>
+                        <a className="dropdown-item" href="#">
+                          Profile
+                        </a>
+                      </li>
+                      <li>
+                        <a className="dropdown-item" href="#">
+                          logOut
+                        </a>
+                      </li>
+                    </ul>
                   </div>
                 </div>
               </li>
             </ul>
           </div>
           <div className="mx-lg-3 ">
+            {/* <button onClick={toggletheme} className="border-0 bg-transparent ">
+              f
+            </button> */}
             <label htmlFor="switch-dark-theme" className="toggle-dark-theme">
```

Comment:
We might extract this into its own component and then use it here. To make the code easier to read. It is generally not recommended to embed svg code in high-level components.

<ToggleSwitch/>
and this component will handle all the logic we need.